### ISSUE TITLE
Unknown error without translation: en.vagrant.errors.winrm_bad_exit_status 

### ISSUE THREAD

**Issue by hh** [WinRb/vagrant-windows#15]
Beyond adding a translation, I'm still unsure what this error is.

```
[windows7] Running provisioner: Vagrant::Provisioners::ChefSolo...
#< CLIXML
<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><S S="Error">Resolve-Path : Cannot find path
 'C:\tmp\vagrant-chef-1\chef-solo-1\cookbooks' b_x000D__x000A_</S><S S="Error">ecause it does not exist._x000D__x000A_</S><S
 S="Error">At line:7 char:15_x000D__x000A_</S><S S="Error">+   Resolve-Path &lt;&lt;&lt;&lt;  $path| Out-Null;_x000D__x000A_
</S><S S="Error">    + CategoryInfo          : ObjectNotFound: (C:\tmp\vagrant-...olo-1\cookboo _x000D__x000A_</S><S S="Erro
r">   ks:String) [Resolve-Path], ItemNotFoundException_x000D__x000A_</S><S S="Error">    + FullyQualifiedErrorId : PathNotFo
und,Microsoft.PowerShell.Commands.Resol _x000D__x000A_</S><S S="Error">   vePathCommand_x000D__x000A_</S><S S="Error"> _x000
D__x000A_</S></Objs>                                                                                                       
C:\opscode\chef\bin\chef-solo.bat
[windows7] Generating chef JSON and uploading...
#< CLIXML
<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><S S="Error">Exception calling "WriteAllByte
s" with "2" argument(s): "Could not find a part _x000D__x000A_</S><S S="Error">of the path 'C:\tmp\vagrant-chef-1\dna.json'.
"_x000D__x000A_</S><S S="Error">At line:28 char:42_x000D__x000A_</S><S S="Error">+           [System.IO.File]::WriteAllBytes
 &lt;&lt;&lt;&lt; ($new_file,$bytes)_x000D__x000A_</S><S S="Error">    + CategoryInfo          : NotSpecified: (:) [], Metho
dInvocationException_x000D__x000A_</S><S S="Error">    + FullyQualifiedErrorId : DotNetMethodException_x000D__x000A_</S><S S
="Error"> _x000D__x000A_</S></Objs>                                                                                        
translation missing: en.vagrant.errors.winrm_bad_exit_status
```

**Comment by pmorton** [comment]
So I can see what is happening, but I don't have enough information on why. It looks like it is having trouble finding c:\tmp\vagrant-chef-1\chef-solo-1\coolbooks. I think that there are two things that need to be resolved. The error needs to be caught and indicate a non-zero exit and second I would like to find out why it could not resolve this path. Any ideas why this path might not exist on your vm?

**Comment by pmorton** [comment]
@hh this was fixed in 1.0. https://github.com/WinRb/vagrant-windows/commit/f414a35dd2c677e959c96d77b9970272b6d041c2.
